cwms-radar-client/src/main/java/mil/army/usace/hec/cwms/radar/client/NoDataFoundException.java

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright (c) 2021
+ * United States Army Corps of Engineers - Hydrologic Engineering Center (USACE/HEC)
+ * All Rights Reserved.  USACE PROPRIETARY/CONFIDENTIAL.
+ * Source may not be released without written approval from HEC
+ */
+
+package mil.army.usace.hec.cwms.radar.client;
+
+import java.io.IOException;
+
+public class NoDataFoundException extends IOException {
+
+    public NoDataFoundException(String message) {
```

Comment:
add time window?

Reply:
that should come back in the error details. Right now I don't have a working example of that that would look like because of https://github.com/USACE/cwms-radar-api/issues/80

Reply:
I can't do that until I have an example. I can log a Jira ticket to follow up though.

Reply:
https://jira.hecdev.net/browse/CWMS-677